### DISCUSSION TITLE
Enemy: Hardlightfoot Hare

### DISCUSSION THREAD

**Discussion by ntseng** [Imaginary-Horizons-Productions/Prophets-of-the-Labyrinth#279] [Ideas]
Futuristic (hardlight) rabbit (lucky)
Labyrinth: Zoo of Chimeras
Element: Light

**Comment by ntseng** [comment]
Is skittish and changes move to a defensive option if it's being targeted?

**Reply by doubledh** [reply]
I mean, unless it's in a pack of nonhares, it's _going_ to be targeted sooner than later. Under current implementations we'd have to have a double-sided move for that, that branches during resolution and has a name that implies either/both fight or flight (else the move predict'd look funny)

**Reply by ntseng** [reply]
That's fair, I'm not sure there's any non-targeting damage in the game yet.

"Fight or Flight" seems like a reasonable name for the move. The intention with the double-sided move is to add "when I do X, Hare does Y" as learnable information that assists the players in manipulating combat.
